app/pb_kits/playbook/pb_image/_image.scss

```diff
@@ -1,15 +1,42 @@
-.pb_image_kit {
-	max-width: 100%;
-	max-height: 100%;
-
-	.blur_up {
-		-webkit-filter: blur(6px);
-		filter: blur(6px);
-		transition: filter 550ms, -webkit-filter 550ms;
-	}
-
-	.blur_up.lazyloaded {
-		-webkit-filter: blur(0);
-		filter: blur(0);
-	}
+@import "../tokens/border_radius";
+@import "../tokens/colors";
+@import "../pb_image/image";
+
+$image-sizes: (
+  "xs": 60px,
+  "sm": 80px,
+  "md": 100px,
+  "lg": 120px,
+  "xl": 140px,
+);
+
+[class^=pb_image_kit] {
+  position: relative;
+
+  @each $name, $size in $image-sizes {
+    &[class*=_#{$name}] {
+      width: $size;
+      height: $size;
+      object-fit: cover;
+      position: relative;
+      flex-shrink: 0;
+      flex-grow: 0;
+      flex-basis: $size;
+    }
+
+    &[class*= rounded] {
+      border-radius: $border-rad-heaviest;
```

Comment:
Should there be a different, corresponding border radius for each `size` option?

Reply:
after talking with the team, we decided there's visually not a significant difference and that for now we will keep it as the one border radius. it's a boolean now, but if we wanted to as a future endeavor it can be changed